### DISCUSSION TITLE
Implement google SSO in seqr application

### DISCUSSION THREAD

**Discussion by nshaikh-aligntech** [broadinstitute/seqr#2877] [Q&A]
Hello,

Just wanted to know if we can implement google SSO in seqr application.
If yes, could you please let me know the process or documentation if available.

Thanks & Regards

**Comment by hanars** [comment]
Yes, google SSO is supported, assuming you have a google cloud project with oauth set up. You need to set the `SOCIAL_AUTH_GOOGLE_OAUTH2_CLIENT_ID` and `SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET` environment variables in your seqr configuration, and you can get these values from the Oath configuration you set up in your google project
